lib/active_merchant/billing/gateways/authorize_net.rb

```diff
@@ -710,6 +712,16 @@ def add_duty_fields(xml, options)
         end
       end
 
+      def add_surcharge_fields(xml, options)
+        surcharge = options[:surcharge] if options[:surcharge]
+        if surcharge.is_a?(Hash)
+          xml.surcharge do
+            xml.amount(amount(surcharge[:amount].to_i)) if surcharge[:amount]
```

Comment:
I think the `to_i` might be unnecessary hear since the `money_format` of this gateway is `dollars` 🤔

Reply:
Hmm now that you mention it, I think that would be fine. Though if both fields are mandatory, we'll not that in the documentation